openslides/motions/static/js/motions/diff.js

```diff
@@ -842,13 +849,11 @@ angular.module('OpenSlidesApp.motions.diff', ['OpenSlidesApp.motions.lineNumberi
 
             if (out.n.length === 0) {
                 for (i = 0; i < out.o.length; i++) {
-                    //this._outputcharcode('del', out.o[i]);
```

Comment:
My change removes this line. If I remove this changing line, this will result in this line to remain in the file :)